### DISCUSSION TITLE
Where can I place umlaut keys?

### DISCUSSION THREAD

**Discussion by JonathanPartain** [manna-harbour/miryoku#206] [Q&A] [answered]
I am interested in giving miryoku a go, but i am unsure where best to position the "å", "ä" and "ö" keys that I regularly use when typing. This also comes with the drawback of symbols not being the same as the US layout. Does anyone have a layout or ideas for this "nordic" type layout?

**Comment by third774** [comment]
First easy thought is to maybe bind them to "meh" and/or "hyper" shortcuts with software in the OS? Would suck to lose those shortcuts though.

**Comment by arasch-saffari** [comment]
Hey, the easy solution is to install the Eurkey keyboard layout on your Mac / Linux / Windows. 

https://eurkey.steffen.bruentjen.eu/?lang=de

With this installed you have the umlaut symbols available and can reach them with Alt + a / o / u / s. The Euro symbol, ẞ and many other symbols are also available.

**Reply by blueyed** [reply]
> reach them with Alt + a / o / u / s

Via AltGr, no?

**Reply by manna-harbour** [reply]
Yes, that should be AltGr. It's not shown in the diagrams, but AltGr is available on the key below Alt on both sides.

**Comment by manna-harbour** [comment, marked as answer]
As @arasch-saffari mentioned, switching to EurKEY is a good solution.

Also see https://github.com/manna-harbour/miryoku/discussions/53#discussioncomment-1761553 and https://github.com/manna-harbour/miryoku/discussions/197#discussioncomment-5240943.
